media-types.md

```diff
@@ -47,3 +47,5 @@ The following figure shows how the above media types reference each other:
 
 [Descriptors](descriptor.md) are used for all references.
 The manifest list being a "fat manifest" references one or more image manifests per target platform. An image manifest references exactly one target configuration and possibly many layers.
+
+[JSON]: http://json.org/
```

Comment:
OK, fine, I asked the wrong question :/. Let me try again: why is it here? (I don't see any JSON references in this document)